README.md

```diff
@@ -22,12 +22,19 @@
 is able to cache different patterns (class, object, output, etc) using different
 storage adapters (DB, File, Memcache, etc).
 
+## Documentation
+
+- [Quickstart and introduction  to laminas](https://docs.laminas.dev/laminas-cache/v3/pattern/intro/)
```

Comment:
@froschdesign is it somehow possible to have some kind of `latest` path so we do not need to update the README whenever we create new majors?

Reply:
@boesing 
_Sorry for the delay, I am on holiday._

There is no _special_ path like `latest` therefore we need an automatic link checker before to prevent wrong or outdated links.
See: https://github.com/laminas/laminas-continuous-integration-action/issues/34